<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trippopup</class>
 <widget class="QDialog" name="trippopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="spentList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="routeList">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalDistanceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total distance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalSpentLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total spent:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Money spent per college</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Route</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
